nengo_gui/static/hotkeys.js

```diff
@@ -86,6 +86,19 @@ Nengo.Hotkeys = function () {
                 Nengo.ace.update_trigger = true;
                 ev.preventDefault();
             }
+            // toggle auto-hide of console with TODO: pick a good shortcut
+            if (ctrl && ev.shiftKey && key == 'h') {
+                Nengo.ace.auto_hide_console = !Nengo.ace.auto_hide_console;
+                if (Nengo.ace.auto_hide_console == false) {
+                    Nengo.ace.console_height = document.getElementById('vmiddle').clientHeight / 4.0;
```

Comment:
Hmm...  I think I'd be more tempted to have that height saved as a config option, so that it's persistent across runs.  So wherever you've moved the console height to stays (as it's a very screen-dependent operation).